### PR TITLE
Created a registeration page and edit profile page

### LINKED ISSUE #15: Create UI for User registration
## Description:

As a user, I want to be able to register so that I can log in whenever I want to create a post for getting a referral. Registration is the first step at the user's end to be able to get the referral. 
The details needed should be:
1. First Name
2. Last Name
3. Current Location
4. Current Company Name
5. Current Position Title
6. University
7. Years of Experience
8. Email Address
9. Password

After the user fills in the right information there should be a register button so that all the information gets passed to the database and then the user can log in with the set of credentials. 

## Acceptance Criteria:
-	[x] The registration page should be accessible from the “/register” path.
-	[x] The page must have a registration form.
-	[x] The login form should have a First name address field with the string data type.
-	[x] The login form should have a Last name address field with the string data type.
-	[x] The login form should have a Current Location address field with the string data type.
-	[x] The login form should have a Current Company Name field with the string data type.
-	[x] The login form should have a University field with the string data type.
-	[x] The login form should have a Years of Experience dropdown with the number data type.
-	[x] The login form should have a Password field with the string data type.
-	[x] There should be a back-to-login form link below the login form.

### LINKED ISSUE #19: Create UI for edit profile
## Description:

As a user, I want to be able to edit the profile. If by mistake the user enters the wrong first name or last name, the user should be able to change. If the user moves to another location, he/ she should be able to update. If the user changes the company or the user's title has changed, he/ she should be able to update the same by editing his/ her profile. Similarly, the years of experience, email address, and password should also be editable. 
The editable details include:
1. First Name
2. Last Name
3. Current Location
4. Current Company Name
5. Current Position Title
6. University
7. Years of Experience
8. Email Address
9. Password

After the user fills in the information he/ she needs to update then there should be a button that says save changes.

As an user, I want a navbar to be visible on this page's top with the name of the website, home page button (#17), requests button, edit profile page button, a green post button, and a logout button.

The dummy page created in #7 for fetching cards data, can be moved to "/home" path and fetch Posts data via API in #18 .

## Acceptance Criteria:
-	[x] The edit profile page should be accessible from the “/edit-profile” path.
-	[x] The form should be prefilled with the previous data and should be editable. 
-	[x] The page must have the same form as the registration form and the form fields should be editable.
- [x] Create a navbar component and display the navbar at the top of the page.
- [x] Move dummy page to "/home" and fetch Posts data.

### PR DESCRIPTION
**Pull request by bahetianjali**
## Acceptance Criteria for edit profile:
-	[x] The edit profile page should be accessible from the “/edit-profile” path.
-	[x] The form should be prefilled with the previous data and should be editable. 
-	[x] The page must have the same form as the registration form and the form fields should be editable.
- [x] Create a navbar component and display the navbar at the top of the page.

## Acceptance Criteria for user registeration:
-	[x] The registration page should be accessible from the “/register” path.
-	[x] The page must have a registration form.
-	[x] The login form should have a First name address field with the string data type.
-	[x] The login form should have a Last name address field with the string data type.
-	[x] The login form should have a Current Location address field with the string data type.
-	[x] The login form should have a Current Company Name field with the string data type.
-	[x] The login form should have a University field with the string data type.
-	[x] The login form should have a Years of Experience dropdown with the number data type.
-	[x] The login form should have a Password field with the string data type.
-	[x] There should be a back-to-login form link below the login form.

### CODE REVIEW

File: client/src/components/Register.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+              <Grid item xs>
+                <Link component={RouterLink} to="/login" variant="body2">
+                  Back to login?
+                </Link>
+              </Grid>
+              <Grid item>
+                <Link component={RouterLink} to="/forgot-password" variant="body2">
+                  Forgot Password?
+                </Link>
+              </Grid>
```
Comment (haxxorsid) [inline client/src/components/Register.tsx]: Remove these 5 lines.

**Review by haxxorsid — CHANGES_REQUESTED** [review]

**Comment by bahetianjali** [comment]
Done with the changes.

**Review by haxxorsid — APPROVED** [review]
Looks good. approved.

**Review by shashank136 — APPROVED** [review]
Approved. Please, push the code.

**Review by Vineetkhadloya — APPROVED** [review]
Approved !!

**Pull request merged**